src/store/modules/order/index.ts

```diff
@@ -0,0 +1,21 @@
+import actions from './actions'
+import getters from './getters'
+import mutations from './mutations'
+import { Module } from 'vuex'
+import OrderState from './OrderState'
+import RootState from '@/store/RootState'
+
+const orderModule: Module <OrderState, RootState> = {
+  namespaced: true,
+  state: {
+    completedOrders: {
+      list: {},
+      total: 0
+    },
+    storedOrders: 0
```

Comment:
Why we have declared storedOrders state here?

Reply:
will change it.